README.md

```diff
@@ -10,8 +10,42 @@ Sails.js/Waterline adapter for MongoDB.
 > As an adapter, this module implements a set of declarative interfaces, conventions, and best-practices for integrating with Mongo databases.
 > Strict adherence to an adapter specification enables the (re)use of built-in generic test suites, standardized documentation, reasonable expectations around the API for your users, and overall, a more pleasant development experience for everyone.
 
+# BREAKING CHANGES
 
-## Installation
+## MongoDB Driver
+From `sails-mongo` version 2.0.0 and above, the adapter uses a new mongodb driver. These driver changes the way it handles connections,
+and implements the concept of [MongoClient](http://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html).
+
+Because of that, `manager` now returns MongoClient, instead of just a connection.
+
+This enables a lot more flexibility and allows the use of the latest MongoDb improvements, like (ClientSession)[http://mongodb.github.io/node-mongodb-native/3.2/api/ClientSession.html],
+and with it, transactions, change streams, and other new features.
+
+## Access to Database object
+If you need to get the database, you have to call the [`db`](http://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html#db) method on the manager (MongoClient).
+
+## Configuration options
+The 0.x and 1.x versions of the adapter use (MongoDb 2.2.x connection options)[http://mongodb.github.io/node-mongodb-native/2.2/api/MongoClient.html#.connect].
+Since the 2.x versions of the adapter, it uses (MongoDb 3.2.x connection options)[http://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html#.connect].
+
+New/updated options:
+ - `authMechanism`: Mechanism for authentication: MDEFAULT, GSSAPI, PLAIN, MONGODB-X509, or SCRAM-SHA-1
+ - `auto_reconnect`: Defaults to `true`. `autoReconnect` still exists, and it takes precedence over the new `auto_reconnect` (https://github.com/mongodb/node-mongodb-native/blob/36c591d7b0a1ee4ad593d1352b9d7eb4e1282afd/lib/topologies/mongos.js#L141)
```

Comment:
This seems a little weird-- is it documented on mongo's side `auto_reconnect` (snake case) now even though all the other options are camel case?  If so, I suppose we have to change it then.  But since we're doing a major version anyway, I'd rather just go ahead and completely remove support for `autoReconnect` at the same time.  That'll make this easier to maintain.  (We could also have a special "autoReconnect is now auto_reconnect" error message as well, but I'm ambivalent about that in this case, since you've already documented it)